test/e2e/select-action.test.js

```diff
@@ -0,0 +1,24 @@
+const { assert, driver } = require('vl-ui-core').Test.Setup;
+const VlMapSelectActionPage = require('./pages/vl-map-select-action.page');
+
+describe('vl-map-select-action', async() => {
+    const vlMapPage = new VlMapSelectActionPage(driver);
+
+    before(() => {
+        return vlMapPage.load();
+    });
+
+    it('Als gebruiker kan ik het verschil zien tussen een select action met cluster en zonder', async () => {
+        const clusteredSelectAction = await vlMapPage.getClusteredSelectAction();
+        const selectAction = await vlMapPage.getSelectAction();
+
+        await assert.eventually.isTrue(clusteredSelectAction.isClustered());
+        await assert.eventually.isFalse(selectAction.isClustered());
+    });
+
+    it('Als gebruiker zie ik dat de select actie actief staat', async () => {
+        const selectAction = await vlMapPage.getSelectAction();
+
+        await assert.eventually.isTrue(selectAction.isActive());
+    });
+});
```

Comment:
Ik zou de testen uitbreiden met de mogelijkheid om een feature te selecteren en om de geselecteerde features op te vragen.

Reply:
Had ik ook eerst geprobeerd, maar helaas niet gevonden. Ik had geprobeerd met mouse actions move to en dan de pixel uit de coördinaat, maar geen effect.

Reply:
check daarvoor ook altijd de testen van impact of zendantennes of ... wij hadden in de verschillende projecten al redelijk wat